core/commands/storage.go

```diff
@@ -1306,7 +1310,20 @@ var storageChallengeCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
 		Tagline: "Interact with storage challenge requests and responses.",
 		ShortDescription: `
-These commands contain both client-side and host-side challenge functions.`,
+These commands contain both client-side and host-side challenge functions.
+
+btfs storage challenge request <peer-id> <contract-id> <file-hash> <shard-hash> <chunk-index> <nonce>
+btfs storage challenge response <contract-id> <file-hash> <shard-hash> <chunk-index> <nonce>
+
+Where 
+
+  <peer-id>     - Host Peer ID to send challenge requests.
+  <contract-id> - Contract ID associated with the challenge requests.
+  <file-hash>   - File root multihash for the data stored at this host.
+  <shard-hash>  - Shard multihash for the data stored at this host.
+  <chunk-index> - Chunk index for this challenge. Chunks available on this host include root + metadata + shard chunks.
+  <nonce>       - Nonce for this challenge. A random UUIDv4 string.
```

Comment:
These are auto-populated options that can be modified under `storageChallengeResponseCmd` instead of duplicating at the upper command. The upper command usually shows higher level summary.

Reply:
@taiyangc I have removed the duplication at the upper command.